master-firmware/src/main.c

```diff
@@ -228,25 +228,6 @@ int main(void)
                         MAX_NB_BUS_ENUMERATOR_ENTRIES);
 
 
-    static __attribute__((section(".ccm"))) motor_driver_t motor_driver_buffer[MAX_NB_MOTOR_DRIVERS];
```

Comment:
We still need this code ! ;)

Reply:
It's back, with other thing, reverted the commit where we "disable some stuff"